### PR TITLE
Visitors can also tell the name and if they want to sit with peers

### PR DESCRIPTION
**Pull request by offcast-dev**
PT: https://www.pivotaltracker.com/story/show/178412348
### User Story
```
As a user
In order to receive a better qualified service
I want to provide some additional info about my inquiry
```
### Changes Proposed
- Add two new questions to wizard
- refactors radio button to take values as props
- adds store assertions to wizard testing
### What learned
- how to assert Redux store
- possible difficulties when making custom components

### CODE REVIEW

**Review by oliverochman — APPROVED** [review]
LGTM

File: cypress/integration/userCanSendInquiry.feature.js [hunk 1]
```diff
@@ -38,6 +30,57 @@ describe('User can send inquiry', () => {
...
+      cy.window()
+        .its('store')
+        .invoke('getState')
+        .its('formData.company')
+        .should('equal', 'Cyberdyne Systems')
+      cy.window()
+        .its('store')
+        .invoke('getState')
+        .its('formData.peers')
+        .should('equal', 'Yes')
```
Comment (1CIM) [inline cypress/integration/userCanSendInquiry.feature.js]: is there any possibility to make this dry?

**Review by arunbhalli — APPROVED** [review]

File: src/components/Inquiry/InquiryCompanyName.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const InquiryCompanyName = () => {
+  const { t } = useTranslation()
+
+  return (
+    <div data-cy='company-name-container'>
+      <Question text={t('question.name')} />
+      <Answer
+        text={t('answer.name')}
+        type='text'
+        placeholder='my company'
```
Comment (1CIM) [inline src/components/Inquiry/InquiryCompanyName.jsx]: placeholder not translatable?

File: src/locales/se.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ const se = {
       email: 'Hur kan vi kontakta dig?',
       size: 'Hur stort är ditt team?',
       officeType: 'Vilken typ av kontor letar du efter?',
+      name: 'Vad är namnet på erat företag?',
+      peer: 'Whould you like to share office with peer companies?'
```
Comment (1CIM) [inline src/locales/se.js]: don't look like Swedish to me ;)

File: cypress/integration/userCanSendInquiry.feature.js (hunk 1 above)
Comment (DavDan1) [reply cypress/integration/userCanSendInquiry.feature.js]: one sec...

File: src/components/Inquiry/InquiryCompanyName.jsx (hunk 2 above)
Comment (DavDan1) [reply src/components/Inquiry/InquiryCompanyName.jsx]: @1CIM  whats up with footer ?

File: cypress/integration/userCanSendInquiry.feature.js (hunk 1 above)
Comment (offcast-dev) [reply cypress/integration/userCanSendInquiry.feature.js]: fixed

File: src/locales/se.js (hunk 3 above)
Comment (offcast-dev) [reply src/locales/se.js]: good spot

File: src/components/Inquiry/InquiryCompanyName.jsx (hunk 2 above)
Comment (1CIM) [reply src/components/Inquiry/InquiryCompanyName.jsx]: @DavDan1 Wut?

File: src/components/Inquiry/Answer.jsx [hunk 4]
```diff
@@ -5,13 +5,23 @@ import store from '../../state/store/configureStore'
...
+  text,
+  type,
+  placeholder,
+  questionKey,
+  label_1,
+  label_2,
+  dataCy_1,
+  dataCy_2,
+  value_1,
+  value_2,
```
Comment (LucasKnudsen) [inline src/components/Inquiry/Answer.jsx]: Could these be stored in arrays / objects, just to to get rid of a bit of em?

File: src/components/custom/CustomRadioButton.jsx [hunk 5]
```diff
@@ -1,34 +1,44 @@
...
+const CustomRadioButton = ({
+  label_1,
+  label_2,
+  dataCy_1,
+  dataCy_2,
+  disabled,
+  onChange,
+  radio_value,
+  value_1,
+  value_2,
```
Comment (LucasKnudsen) [inline src/components/custom/CustomRadioButton.jsx]: And is there a way for some of these values to be used for the same things, or is that already optimized?

**Review by LucasKnudsen — APPROVED** [review]

File: src/components/Inquiry/InquiryCompanyName.jsx (hunk 2 above)
Comment (offcast-dev) [reply src/components/Inquiry/InquiryCompanyName.jsx]: it is now

File: src/components/Inquiry/Answer.jsx (hunk 4 above)
Comment (offcast-dev) [reply src/components/Inquiry/Answer.jsx]: refactered

File: src/components/custom/CustomRadioButton.jsx (hunk 5 above)
Comment (offcast-dev) [reply src/components/custom/CustomRadioButton.jsx]: I think we can use value and label, but I thought that we may want some flexibility. Maybe premature optimization, but also it is more clear to assign value and label separately.

**Pull request merged**